.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "0.12"
+  - "5.0"
```

Comment:
> p.s. Don't use .0.

It's definitely out of the topic, but 0.x would be widely used in [enterprise company title] until Q1-Q2/2016.